devenv/run_demo.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+###############################################################################
+# --- This script installs the development version of b2share2 ---
+###############################################################################
+
+### The following commands must be run beforehand
+# brew install python --framework --universal
+# pip install virtualenv
+# pip install virtualenvwrapper
+
+source /usr/local/bin/virtualenvwrapper.sh
+
+export VIRTUALENV_NAME='b2share-evolution'
+export DB_NAME='b2share-evolution'
+export MACHINE_NAME=b2share
+
+#deactivate if we're already in a virtual env
+deactivate
+
+workon $VIRTUALENV_NAME
+if [ $? -ne 0 ]; then
+	echo; echo "### Make virtual env"
+	mkvirtualenv $VIRTUALENV_NAME
+	workon $VIRTUALENV_NAME
+	cdvirtualenv && mkdir src
+fi
+
+docker-machine start $MACHINE_NAME
+if [ $? -ne 0 ]; then
+	echo; echo "### Create docker machine for b2share"
+	docker-machine create -d virtualbox $MACHINE_NAME
+	docker-machine start $MACHINE_NAME
+fi
+eval $(docker-machine env $MACHINE_NAME)
+DOCKER_IP=`docker-machine ip $MACHINE_NAME`
+
+
+cdvirtualenv src
+if [ ! -d ./ui-frontend ]; then
+	echo; echo "### Clone b2share UI"
+	git clone git@github.com:EUDAT-B2SHARE/ui-frontend.git
+	cd ui-frontend
+	npm install
+fi
+
+
+cdvirtualenv src
+if [ ! -d ./b2share ]; then
+	echo; echo "### Clone b2share"
+	git clone git@github.com:EUDAT-B2SHARE/b2share.git --branch evolution
```

Comment:
The ui-frontend and b2share repo should be tagged with the same tag for beta released. So that we can know which versions are compatible.
We can change that once we have stable releases and don't need demo scripts.